XpringKit/XRPPayment.swift

```diff
@@ -1,6 +1,8 @@
 import Foundation
 
 // TODO(keefertaylor): Modify this object to use X-Address format.
```

Comment:
Can you place this TODO after the class comment? 

So it reads:
<summary>
<random TODOs>
<code>